Geometry/GEMGeometryBuilder/plugins/ME0GeometryValidate.cc

```diff
@@ -102,17 +102,17 @@ class ME0GeometryValidate : public one::EDAnalyzer<> {
 };
 
 ME0GeometryValidate::ME0GeometryValidate(const edm::ParameterSet& iConfig)
-    : infileName_(
-          iConfig.getUntrackedParameter<string>("infileName", "cmsRecoGeom-2026.root")),  //it was cmsGeom10.root
+    : tokGeom_{esConsumes<ME0Geometry, MuonGeometryRecord>(edm::ESInputTag{})},
+      infileName_(iConfig.getUntrackedParameter<string>("infileName", "cmsRecoGeom-2026.root")),
       outfileName_(iConfig.getUntrackedParameter<string>("outfileName", "validateME0Geometry.root")),
       tolerance_(iConfig.getUntrackedParameter<int>("tolerance", 6)) {
   fwGeometry_.loadMap(infileName_.c_str());
   outFile_ = TFile::Open(outfileName_.c_str(), "RECREATE");
 }
 
 void ME0GeometryValidate::analyze(const edm::Event& event, const edm::EventSetup& eventSetup) {
-  eventSetup.get<MuonGeometryRecord>().get(me0Geometry_);
-  if (me0Geometry_.isValid()) {
+  me0Geometry_ = &eventSetup.getData(tokGeom_);
+  if (me0Geometry_ != nullptr) {
```

Comment:
This is a change in functionality. The `EventSetup::getData()` throws an exception if the product is not available. In order to check the availability of the product, `EventSetup::getHandle()`, that returns `ESHandle`, should be used .